core/src/main/kotlin/api/OrganizationsRoute.kt

```diff
@@ -78,15 +80,15 @@ fun Route.organizations() = route("organizations") {
     val infrastructureServiceService by inject<InfrastructureServiceService>()
 
     get(getOrganizations) { _ ->
-        requireSuperuser()
-
         val pagingOptions = call.pagingOptions(SortProperty("name", SortDirection.ASCENDING))
 
-        val organizations = organizationService.listOrganizations(pagingOptions.mapToModel())
-        val pagedResponse = PagedResponse(
-            organizations.map { it.mapToApi() },
-            pagingOptions
-        )
+        val organizations = organizationService
+            .listOrganizations(pagingOptions.copy(limit = null, offset = null).mapToModel())
+            .filter { hasPermission(it.id, OrganizationPermission.READ) }
```

Comment:
Just to clarify: If a user is in the writers group, will they also have the role to read? I couldn't find that in the code quickly.

Reply:
Okay found it already by myself. The writers group has the write and read permission roles.